account_invoice_refund_option/__manifest__.py

```diff
@@ -0,0 +1,20 @@
+# Copyright 2016 Jairo Llopis <jairo.llopis@tecnativa.com>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl).
+{
+    "name": "Create Refund Invoice",
+    "summary": "Allows you to create directly a refund without starting from "
+               "an invoice",
+    "version": "11.0.1.0.0",
+    "category": "Accounting",
+    "website": "https://www.tecnativa.com/",
```

Comment:
@ThomasBinsfeld Replace URL as per [this](https://github.com/odoo/odoo/blob/11.0/addons/hr_recruitment/__manifest__.py#L11)